modules/apps/frontend-css/frontend-css-custom-properties-sample-web/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,14 @@
+aspect-ratios=Aspect Ratios
```

Comment:
Are all these the same labels we added in the `theme-classic` module?

Reply:
I'm not sure if these are the same but they are at least similar for sure

Reply:
Yeah, probably, and if they're not, we probably want them to be :)

See https://github.com/liferay-frontend/liferay-portal/pull/279#discussion_r476461481